docs/pages/includes/database-access/azure-troubleshooting.mdx

```diff
@@ -0,0 +1,18 @@
+### No credential providers error
+
+If you see the error `DefaultAzureCredential: failed to acquire a token.` in Database Service logs then Teleport
+is not detecting the required credentials to connect to the Azure SDK. Check whether
+the credentials have been applied in the machine running the Teleport Database Service.
+Refer to [Azure SDK Authorization](https://docs.microsoft.com/en-us/azure/developer/go/azure-sdk-authorization)
+for more information.
```

Comment:
Assuming once the credentials is fixed, there is no need to restart the service right?

Reply:
Hmm I'll have to check that. I think it picks up the new credentials, but it may not since we cache the token

Reply:
It does need to be restarted actually, I'll add that to the doc